src/main/java/de/tum/in/www1/artemis/service/AssessmentService.java

```diff
@@ -92,8 +92,8 @@ public AssessmentService(ComplaintResponseService complaintResponseService, Comp
      * @param assessmentUpdate the assessment update containing a ComplaintResponse and the updated Feedback list
      * @return the updated Result
      */
-    public Result updateAssessmentAfterComplaint(Result originalResult, Exercise exercise, AssessmentUpdate assessmentUpdate) {
-        if (assessmentUpdate.feedbacks() == null || assessmentUpdate.complaintResponse() == null) {
+    public Result updateAssessmentAfterComplaint(Result originalResult, Exercise exercise, AssessmentUpdateBaseDTO assessmentUpdate) {
+        if (assessmentUpdate.complaintResponse() == null) {
             throw new BadRequestAlertException("Feedbacks and complaint response must not be null.", "AssessmentUpdate", "notnull");
         }
```

Comment:
Why was the check for feedbacks removed? If it was intentional please update the exception as follows:
```suggestion
    public Result updateAssessmentAfterComplaint(Result originalResult, Exercise exercise, AssessmentUpdateBaseDTO assessmentUpdate) {
        if (assessmentUpdate.complaintResponse() == null) {
            throw new BadRequestAlertException("Complaint response must not be null.", "AssessmentUpdate", "notnull");
        }
```